pkg/apis/core/toleration_test.go

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package core
+
+import "testing"
+
+func TestMatchToleration(t *testing.T) {
```

Comment:
`MatchToleration()` checks that all fields of `Toleration` are the same, except `TolerationSeconds`. It ignores `TolerationSeconds`. Please add a case to verify that `MatchToleration()` returns true even when `TolerationSeconds` of two `Toleration`s are not equal.

Reply:
Done, thanks @bsalamat